packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -99,11 +107,21 @@ export default class TableResize implements EditorPlugin {
             replaceSelection: false,
             position: ContentPosition.Outside,
         });
+
+        this.tableResizerContainer = this.editor.getDocument().createElement('div');
+        this.editor.insertNode(this.tableResizerContainer, {
+            updateCursor: false,
+            insertOnNewLine: false,
+            replaceSelection: false,
+            position: ContentPosition.Outside,
+        });
     }
 
     private removeResizerContainer() {
         this.resizerContainer.parentNode.removeChild(this.resizerContainer);
         this.resizerContainer = null;
+        this.tableResizerContainer.parentNode.removeChild(this.tableResizerContainer);
```

Comment:
Where there need to have a separate container for table resizer? can they share the same one?

Reply:
It could happen that we have both inserter and table resizer shown at the same time (same behavior in Word). If we share the same container, it would make thing a little bit complicated, let's say, it is showing both inserter and table resizer, now the user moves mouse to the middle of the table, so we want to remove the inserter from the container but still keep the table resizer. I feel like keeping them in separate containers will make it less bug prone.